_tour/default-parameter-values.md

```diff
@@ -62,3 +62,24 @@ public class Main {
 ```
 {% endtab %}
 {% endtabs %}
+
+### Default Parameters for Overloaded Methods
+
+It's important to notice that Scala doesn't allow having two methods with default parameters and with the same name (overloaded).
+
+For example, if you write `def f(a: Int = 1)`, the compiler generates `def f$default$1 = 1`. If you have two overloads with defaults on the same parameter position, then we would need a different naming scheme. In this situation, it is preferred to keep the generated byte-code stable over multiple compiler runs.
+
+An important reason why is to avoid the ambiguity that can be caused due to the existence of default parameters. To illustrate the problem, let's consider the method declarations provided below:
+
+{% tabs default-parameter-values-5 %}
+{% tab 'Scala 2 and 3' for=default-parameter-values-5 %}
+```scala mdoc:fail
+object A {
+  def func(x: Int = 34): Unit
+  def func(y: String = "abc"): Unit
+}
+```
+{% endtab %}
+{% endtabs %}
```

Comment:
This final change is just for consistency
~~~suggestion
{% tabs default-parameter-values-5 class=tabs-scala-version %}
{% tab 'Scala 2' %}
```scala mdoc:fail
object A {
  def func(x: Int = 34): Unit
  def func(y: String = "abc"): Unit
}
```
{% endtab %}
{% tab 'Scala 3' %}
```scala
object A:
  def func(x: Int = 34): Unit
  def func(y: String = "abc"): Unit
```
{% endtab %}
{% endtabs %}
~~~